tests/unit/moduleapi/blockonbackground.tcl

```diff
@@ -0,0 +1,15 @@
+set testmodule [file normalize tests/modules/blockonbackground.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test {SLOWLOG - check that logs commands taking more time than specified including the background time} {
+        r config set slowlog-log-slower-than 100000
+        r ping
+        assert_equal [r slowlog len] 0
+        r block.debug 0 10
+        assert_equal [r slowlog len] 0
+        r block.debug 2 10
```

Comment:
i don't like that long sleep (slows down the whole test suite by a lot).
let's try to find a shorter way to test it.
maybe sleeping 100ms, and making sure that not only it appears in the slowlog, but also that the duration it shows is longer than 100ms?

Reply:
@oranagra followed your review and reduced the sleep to 100ms and effectively checking the ustime and usec_per_call of that blocked call.